docs/web3.contract.rst

```diff
@@ -35,7 +35,13 @@ After ``py-solc-x`` is installed, you will need to install a version of ``solc``
     >>> from solcx import install_solc
     >>> install_solc(version='latest')
 
-You should now be set up to run the contract deployment example below:
+You should now be set up to compile and deploy a contract.
+
+The following example runs through these steps:
+1. Compile Solidity contract into bytecode and an ABI
```

Comment:
Formatting in the docs isn't coming out as intended: https://web3py--3137.org.readthedocs.build/en/3137/web3.contract.html#contract-deployment-example

Looks like you want `#.` instead of the actual numbers: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/lists.html

Reply:
Oops, thanks. Fixed!